ambry-store/src/test/java/com.github.ambry.store/StorageManagerTest.java

```diff
@@ -167,6 +167,56 @@ public void scheduleAndControlCompactionTest() throws Exception {
     shutdownAndAssertStoresInaccessible(storageManager, replicas);
   }
 
+  /**
+   * Test add new BlobStore with given {@link ReplicaId}.
+   */
+  @Test
+  public void addBlobStoreTest() throws Exception {
+    generateConfigs(true);
+    MockDataNodeId localNode = clusterMap.getDataNodes().get(0);
+    List<ReplicaId> localReplicas = clusterMap.getReplicaIds(localNode);
+    int newMountPathIndex = 3;
+    // add new MountPath to local node
+    File f = File.createTempFile("ambry", ".tmp");
+    File mountFile = new File(f.getParent(), "mountpathfile" + 62000 + newMountPathIndex);
```

Comment:
what's the "62000"?

Reply:
Sorry, I should use a local variable to denote this. 62000 is `currentPlainTextPort` used in `MockClusterMap`. I was trying to make it consistent with the setup in `MockClusterMap`. Let me change this and make it clear.